router-tests/testenv/pubsub.go

```diff
@@ -26,8 +29,19 @@ var (
 )
 
 type KafkaData struct {
-	Brokers   []string
-	Container *kafka.KafkaContainer
+	Brokers []string
+}
+
+func getHostPort(resource *dockertest.Resource, id string) string {
```

Comment:
Why is this part of your PR?

Reply:
That's fine but then also update the PR title and description.

Reply:
Done